src/components/flat-table/flat-table.stories.mdx

```diff
@@ -1092,6 +1089,120 @@ remains sticky.
   </Story>
 </Canvas>
 
+In this example of FlatTable with `stickyFooter`, FlatTable has been placed inside of a `Box` with a height
+of `220px` and a column contains a long piece of text that causes the table to overflow horizontally. 
+
+<Canvas>
```

Comment:
I can understand why we want this story here and with Chromatic enabled - to avoid a regression on the earlier bug around this. But I'm not sure it's adding much to the documentation. In my opinion this would be better as a test story - I believe we can still run Chromatic on those.

Reply:
Good thinking. I'll get that sorted 😄